language-plutus-core/untyped-plutus-core/Language/UntypedPlutusCore/Core/Type.hs

```diff
@@ -0,0 +1,60 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE TypeFamilies      #-}
+
+module Language.UntypedPlutusCore.Core.Type where
```

Comment:
Now that we don't have `Type`s it seems a little perverse to put the `Term` definition in a file called `Type.hs`!

Reply:
Yeah, it probably should be `Types.hs`, but I'm trying to be consistent with the structure of Typed Plutus Core.